ts/features/wallet/component/NewMethodAddedNotifier.tsx

```diff
@@ -0,0 +1,73 @@
+import { useBottomSheetModal } from "@gorhom/bottom-sheet";
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import I18n from "../../../i18n";
+import { GlobalState } from "../../../store/reducers/types";
+import { bottomSheetContent } from "../../../utils/bottomSheet";
+import { useNavigationContext } from "../../../utils/hooks/useOnFocus";
+import BancomatInformation from "../bancomat/screen/BancomatInformation";
+import { onboardingBancomatAddedPansSelector } from "../onboarding/bancomat/store/reducers/addedPans";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const newBancomatBottomSheet = () => {
+  const { present, dismiss } = useBottomSheetModal();
+
+  const openModalBox = async () => {
+    const bottomSheetProps = await bottomSheetContent(
+      <BancomatInformation onAddPaymentMethod={dismiss} />,
+      I18n.t("wallet.methods.pagobancomat.name"),
+      500,
+      dismiss
+    );
+    present(bottomSheetProps.content, {
+      ...bottomSheetProps.config
+    });
+  };
+  return { present: openModalBox, dismiss };
+};
+
+/**
+ * Handle the notification for a new payment method added
+ * TODO: add other methods, handle only bancomat atm
+ * @constructor
+ */
+const NewPaymentMethodAddedNotifier = (props: Props) => {
+  const navigation = useNavigationContext();
+  // Save the latest visualized bottom sheet, in order to avoid to show it again if focus changed
+  const [lastNotifiedBancomatHash, setLastNotifiedBancomatHash] = useState<
```

Comment:
Could be a better idea to use an array or a Set?
Considering this scenario:

- add bancomat A, B
- BS is shown

- add a subset of previous addition, ex adding only B
- BS is shown

Is this a desired behavior?
Using an array/set this could be prevented

Reply:
As we have agreed, in this version we keep this functioning to refine it later